test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12605,6 +12671,28 @@ def pbs_version(self):
 
         return self.version
 
+    def delete_vnodes(self):
+        rah = ATTR_rescavail + '.host'
+        rav = ATTR_rescavail + '.vnode'
+        a = {rah: self.hostname, rav: None}
+        try:
+            _vs = self.server.status(HOST, a, id=self.hostname)
+        except PbsStatusError:
+            try:
+                _vs = self.server.status(HOST, a, id=self.shortname)
+            except PbsStatusError as e:
+                if e.msg[0].endswith('Server has no node list'):
+                    _vs = []
+                else:
+                    raise e
+        vs = []
+        for v in _vs:
+            if v[rav].split('.')[0] != v[rah].split('.')[0]:
+                vs.append(v['id'])
+        if len(vs) > 0:
+            self.server.manager(MGR_CMD_DELETE, VNODE, id=vs,
+                                expect=True)
```

Comment:
I thought expect=True only changed what manager() returned.  If you're not concerned about the return value of manager(), do you need to expect=True?